### PR TITLE
- allow md5 and sha1

### PR DESCRIPTION
**Pull request by SSE4**

### CODE REVIEW

**Comment by SSE4** [comment]
@danimtb added tests
P.S. CI error appears to be unrelated:
```
ERROR: The Python ssl extension was not compiled. Missing the OpenSSL lib?
```

File: tests/test_hooks/conan-center/test_conandata.py [hunk 1]
```diff
@@ -60,6 +60,34 @@ def source(self):
...
+                    sources:
+                      "1.69.0":
+                          - url: "url1_1.69.0"
+                            sha256: "sha1_1.69.0"
+                          - url: "url2_1.69.0"
+                            sha256: "sha2_1.69.0"
+                          """)
+        tools.save('conandata.yml', content=conandata)
+        export_output = self.conan(['export', '.', 'name/1.69.0@jgsogo/test'])
+        self.assertNotIn("ERROR: [CONANDATA.YML FORMAT (KB-H030)]", export_output)
```
Comment (danimtb) [inline tests/test_hooks/conan-center/test_conandata.py]: assert conan data format ok?

File: tests/test_hooks/conan-center/test_conandata.py [hunk 2]
```diff
@@ -60,6 +60,34 @@ def source(self):
...
+                    sources:
+                      "1.69.0":
+                          url: "url1.69.0"
+                          sha256: "sha256_1.69.0"
+                          sha1: "md5_1.69.0"
+                          md5: "sha1_1.69.0"
+                          """)
+        tools.save('conandata.yml', content=conandata)
+        export_output = self.conan(['export', '.', 'name/1.69.0@jgsogo/test'])
+        self.assertNotIn("ERROR: [CONANDATA.YML FORMAT (KB-H030)]", export_output)
```
Comment (danimtb) [inline tests/test_hooks/conan-center/test_conandata.py]: same here

Comment (SSE4) [reply tests/test_hooks/conan-center/test_conandata.py]: added

File: tests/test_hooks/conan-center/test_conandata.py (hunk 1 above)
Comment (SSE4) [reply tests/test_hooks/conan-center/test_conandata.py]: added

**Pull request merged**